Filter/ModelFilter.php

```diff
@@ -96,7 +96,14 @@ protected function handleMultiple(ProxyQueryInterface $queryBuilder, $alias, $da
         $parameterName = $this->getNewParameterName($queryBuilder);
 
         if (isset($data['type']) && $data['type'] == EqualType::TYPE_IS_NOT_EQUAL) {
-            $this->applyWhere($queryBuilder, $queryBuilder->expr()->notIn($alias, ':'.$parameterName));
+            $or = $queryBuilder->expr()->orX();
+
+            $or->add($queryBuilder->expr()->notIn($alias, ':'.$parameterName));
+
+            $allAliases = $queryBuilder->getAllAliases();
```

Comment:
Why pick `getAllAliases` over `getRootAliases` here? Is it deliberate? Also, I think you can avoid this one-time variable by writing `current(($queryBuilder->getAllAliases()))`